10-JavaScript/exercises/functions-worksheet.md

```diff
@@ -43,42 +43,25 @@ def output(items)
 end
 ```
 
-6.  
-```ruby
-class Chair
-  def initialize(name)
-    @name = name
-  end
-
-  def name
-    return @name
-  end  
-
-  def weight_in_lbs
-    return 29
-  end
-end
-```
-
 
 ### Part 2
 1.
 Invoke the zombies function.
 ```javascript
-var zombies = function() {
+const zombies = function() {
   return "We like to eat people";
 };
 ```
 
 2.
 Call the `square()` method on the `my_calculator` object.
 ```javascript
-var Calculator = function() {
+const Calculator = function() {
   this.square = function (x) {
```

Comment:
If we've got an example of prototypical objects here, do we want to add a classical object example as well?

Reply:
using a `class`? I'd like to review when this worksheet is assigned to see what we can/should be covering